CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- `serve` now respects the `entrypoint` configured in `polymer.json`.
```

Comment:
I would add what practical effect this has for users. Does this make `polymer serve -o` do the expected thing? Does it make the custom-elements-es5-adapter go into the right file?